mypyc/test-data/irbuild-basic.test

```diff
@@ -3273,7 +3273,7 @@ L2:
 [case testFinalStaticInt]
 from typing import Final
 
-x: Final = 1 + 1
+x: Final = 1 + int()
```

Comment:
The intent of changes like these is to preserve the old test case behavior where constant folding wasn't performed.